packages/react-button/package.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "@fluentui/react-button",
+  "version": "0.1.0",
+  "description": "Fluent UI React Button component.",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test",
+    "update-api": "just-scripts update-api",
+    "update-snapshots": "just-scripts jest -u"
+  },
+  "devDependencies": {
+    "@types/enzyme": "3.10.3",
+    "@types/enzyme-adapter-react-16": "1.0.3",
+    "@types/es6-promise": "0.0.32",
+    "@types/jest": "~24.9.0",
+    "@types/react": "16.8.25",
+    "@types/react-dom": "16.8.4",
+    "@types/react-test-renderer": "^16.0.0",
+    "@types/webpack-env": "1.15.1",
+    "@uifabric/build": "7.0.0",
+    "@uifabric/tslint-rules": "7.2.0",
+    "enzyme": "~3.10.0",
+    "enzyme-adapter-react-16": "^1.15.0",
+    "es6-weak-map": "^2.0.2",
+    "react": "16.8.6",
+    "react-app-polyfill": "~1.0.1",
+    "react-dom": "16.8.6",
+    "react-test-renderer": "^16.3.0"
+  },
+  "dependencies": {
+    "@uifabric/set-version": "7.0.11",
+    "@fluentui/react-compose": "^0.2.11",
+    "@uifabric/react-hooks": "^7.3.1",
+    "@uifabric/utilities": "^7.16.2",
```

nit: Order the packages alphabetically
```suggestion
    "@fluentui/react-compose": "^0.2.11",
    "@uifabric/react-hooks": "^7.3.1",
    "@uifabric/set-version": "7.0.11",
    "@uifabric/utilities": "^7.16.2",
```